source/Irrlicht/CGLTFMeshFileLoader.cpp

```diff
@@ -311,13 +311,16 @@ void CGLTFMeshFileLoader::MeshExtractor::copyTCoords(
  * Required: NO
  * @returns: core::vector2df
 */
-float CGLTFMeshFileLoader::MeshExtractor::getScale() const
+core::vector3df CGLTFMeshFileLoader::MeshExtractor::getScale() const
 {
-	if (m_model.nodes[0].scale.size() > 0) {
-		return static_cast<float>(m_model.nodes[0].scale[0]);
+	core::vector3df buffer{1,1,1};

```

Comment:
```suggestion
	core::vector3df buffer{1.0f,1.0f,1.0f};

```

Reply:
This does not look resolved to me.

Reply:
Fixed in a95861a8487cc847e65a7beffdf5e7c1cc2d95d4